util/config.mjs

```diff
@@ -0,0 +1,9 @@
+export function config() {
```

Comment:
remember to add back `dotenv` with [the `dotenv-safe` replacement](https://github.com/rolodato/dotenv-safe/issues/128#issuecomment-1383176751)

Reply:
6bc67fc